string name
int64 height
int64 width
int8[] int_array
